src/libcore/atomic.rs

```diff
@@ -1089,3 +1090,21 @@ pub fn fence(order: Ordering) {
         }
     }
 }
+
+macro_rules! impl_Debug {
+    ($($t:ident)*) => ($(
+        impl fmt::Debug for $t {
```

Comment:
Trait impls are insta-stable, so it seems kinda useless...?

I feel like I've had this conversation a ton of times and keep forgetting why it might be useful.